src/index.html

```diff
@@ -7,6 +7,423 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <div class="wrapper">
+      <header class="header">
+        <div class="header__logo">
+          <a
+            href="#"
+            class="header__link"
+          >
+            <img
+              src="images/logo.png"
+              alt="#"
+              class="header__picture"
+             >
+          </a>
+          <nav class="nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a href="#" class="nav__link is-active">
+                  Apple
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Samsung
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Смартфоны и телефоны
+                </a>
+              </li>
+              <li class="nav__item">
+                <a
+                  href="#"
+                  class="nav__link"
+                  data-qa="nav-hover"
+                >
+                  Ноутбуки и компьютеры
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Гаджеты
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Планшеты
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Фото
+                </a>
+              </li>
+              <li class="nav__item">
+                <a href="#" class="nav__link">
+                  Видео
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </header>
+      <main class="main">
+        <div class="catalog">
+          <div class="catalog__container">
+            <div class="catalog__row">
+              <div class="card" data-qa="card">
+                <div class="card__picture">
+                  <img
+                    src="./images/imac.jpeg"
+                    alt="imac"
+                    class="card__image"
+                  >
+                </div>
+                <h2 class="card__title">
+                  Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+                </h2>
+                <p class="card__subtitle">
+                  Код товара: 195434
+                </p>
+                <div class="card__rating">
+                  <div class="stars">
```

Comment:
You should add a modifier there like in `stars` task, to change amount of active stars.
And in other cards do the same.
```suggestion
                  <div class="stars stars--4">
```